st2actions/st2actions/config.py

```diff
@@ -71,9 +71,9 @@ def _register_action_runner_opts():
         cfg.IntOpt('max_parallel_actions', default=50,
                    help='Max number of parallel remote SSH actions that should be run.  ' +
                         'Works only with Paramiko SSH runner.'),
-        cfg.BoolOpt('use_ssh_config',
-                    default=False,
-                    help='Use the .ssh/config file. Useful to override ports etc.')
+        cfg.StrOpt('ssh_config_path',
```

Comment:
Oops, my bad, I may have deleted the `use_ssh_config` while sorting out the merge conflicts. Fix on its way.